packages/graph/hash_graph/lib/graph/src/ontology/types/mod.rs

```diff
@@ -64,15 +64,15 @@ impl<T: OntologyType> Persisted<T> {
         }
     }
 
-    pub const fn version_id(&self) -> VersionId {
-        self.version_id
+    pub const fn version_id(&self) -> &VersionId {
+        &self.version_id
```

Comment:
Changed this to return a reference for an easier interface when interacting with `&[&dyn ToSql]`, this way, we simply can call `version_id` without temporarily storing the type.